web/client/components/map/leaflet/MeasurementSupport.jsx

```diff
@@ -105,6 +114,7 @@ const MeasurementSupport = React.createClass({
                 areaMeasureEnabled: this.props.measurement.areaMeasureEnabled,
                 bearingMeasureEnabled: this.props.measurement.bearingMeasureEnabled,
                 geomType: this.props.measurement.geomType,
+                point: null, // handled in onDraw.created
```

Comment:
Indeed, commit amended. BTW I'll have a followup PR which will clean up the code a bit.